homeassistant/components/history/__init__.py

```diff
@@ -36,12 +38,12 @@
 from homeassistant.helpers.typing import ConfigType
 import homeassistant.util.dt as dt_util
 
-# mypy: allow-untyped-defs, no-check-untyped-defs
-
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = "history"
 HISTORY_FILTERS = "history_filters"
+HISTORY_USE_INCLUDE_ORDER = "history_use_include_order"
```

Comment:
Ooof what a dumb option.

Reply:
Once we revamp the history UI, this option can go.

Reply:
I saw it and... 🤦    Talk about backend dev not wanting to do frontend work